app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+# ユーザー情報を編集する
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)